lib/syslogger.rb

```diff
@@ -126,7 +126,7 @@ def max_octets=(max_octets)
   def level=(level)
     level = Logger.const_get(level.to_s.upcase) if level.is_a?(Symbol)
 
-    unless level.is_a?(Fixnum)
+    unless level.is_a?(Integer)
```

Comment:
This change will make this gem Ruby 2.4+ only. You should add this support conditionally. I think the canonical way is checking for class of `0`, i.e. `if 0.class == Integer`